### PR TITLE
MAINT: Fix a few install issues

### LINKED ISSUE #7062: [BUG] Outdated Qt version crashes mne coreg
When using mne coreg, if the Qt version is before 5.10, the gui doesn't work. @larsoner suggested adding an error to detect and throw an error on launch if qt < 5.10 and MacOS.

### LINKED ISSUE #7075: minor issue - install python on window
Hi from Hackhathon Brainthemind Nov 2019 # 2
When following instructions here: 
https://mne.tools/stable/install/mne_python.html#installing-python

1) command lines like "which" may not work in windows but it is fine

2) python -c 'import mne; mne.sys_info()'
window users should use
python -c "import mne; mne.sys_info()"

### PR DESCRIPTION
**Pull request by larsoner**
1. Check PyQt5 version on `sys_info` and all GUI calls, warn if on OSX and Qt<5.10 (as this is problematic), cc @alexrockhill as he had this problem
2. Modernize related compat checks in `mne/gui`
3. Add notes to installation about `which` on Linux -> `where` on Windows
4. Fix quotes in suggested command so that it works on Windows (and still on Linux/macOS)
5. Just verify that `mne` is accessible as a command-line option on Windows machines (tested on Azure)

Closes #7075
Closes #7062

### CODE REVIEW

**Comment by larsoner** [comment]
Okay looks like there is some problem!
```
Script contents:
mne --help
========================== Starting Command Output ===========================
##[command]"C:\windows\system32\cmd.exe" /D /E:ON /V:OFF /S /C "CALL "d:\a\_temp\886d77a5-69c1-45b2-b4f6-ff60249132b5.cmd""
'mne' is not recognized as an internal or external command,
operable program or batch file.
```

File: setup.py [hunk 1]
```diff
@@ -97,4 +97,6 @@ def package_tree(pkgroot):
               op.join('io', 'artemis123', 'resources', '*.csv'),
               op.join('io', 'edf', 'gdf_encodes.txt')
           ]},
-          scripts=['bin/mne'])
+          entry_points={'console_scripts': [
+              'mne = mne.commands.utils:main',
+          ]})
```
Comment (larsoner) [inline setup.py]: @agramfort I deleted `bin/mne` and added this, it seems to be the modern way to specify command entry points. Azure is happy and I can confirm locally on Windows `mne` does not work on 0.19 and on this PR with this change it does. But just wanted to point this out so you could look and not be surprised when `bin` is gone :)

Comment (agramfort) [reply setup.py]: nice !

**Comment by agramfort** [comment]
Thx @larsoner

**Pull request merged**